### PR TITLE
Fixed crash for <1.13 and added blast resistance

### LINKED ISSUE #11: Error extracting data TypeError: Cannot read property 'numeric_id' of undefined
I tried extracting the data from using Burger on 1.12.2 and got an error, any assistance is appreciated
> I ran `npm i` less than a minute before running burger-extractor, so minecraft-data can't be out of date

> I even tried rerunning everything after manually changing the `package.json` to have the most up-to-date version of minecraft-data
```
PS C:\Users\username\Documents\git-projs\data\burger-extractor> node src/index ../output/output.json 1.12.2

 > Make sure to have the latest version of minecraft-data installed
 > Created output directory

  Data extraction started
    Extracting biome data
    Extracting block data
    Extracting entity data
    Extracting item data
    Extracting recipe data
 > Error extracting data TypeError: Cannot read property 'numeric_id' of undefined
TypeError: Cannot read property 'numeric_id' of undefined
    at findItemByName (C:\Users\username\Documents\git-projs\data\burger-extractor\src\extractors\blocks.js:20:29)
    at C:\Users\username\Documents\git-projs\data\burger-extractor\src\extractors\blocks.js:79:44
    at new Promise (<anonymous>)
    at module.exports (C:\Users\username\Documents\git-projs\data\burger-extractor\src\extractors\blocks.js:15:58)
    at C:\Users\username\Documents\git-projs\data\burger-extractor\src\index.js:63:49
    at Array.map (<anonymous>)
    at run (C:\Users\username\Documents\git-projs\data\burger-extractor\src\index.js:63:32)
    at Object.<anonymous> (C:\Users\username\Documents\git-projs\data\burger-extractor\src\index.js:77:1)
    at Module._compile (internal/modules/cjs/loader.js:1251:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1272:10)
```

### PR DESCRIPTION
**Pull request by Moondarker**
Fixes #11
P.S. I have no idea why package.json got edited, probably an auto-ident's doing while it was open

### CODE REVIEW

**Comment by Moondarker** [comment]
Added blast resistance while I'm at it, should help with https://github.com/PrismarineJS/minecraft-data/issues/383 https://github.com/PrismarineJS/mineflayer/issues/1789

**Comment by rom1504** [comment]
Can you run it and pr the result to minecraft data ?
So we can see how it looks

**Comment by Moondarker** [comment]
I can, but it seems like minecraft-data currently has some differences with output of burger-extractor?
i.e. drops, minStateId, maxStateId and states. Is there another post-processing step I'm missing?

**Comment by rom1504** [comment]
There are many extractors https://github.com/PrismarineJS/minecraft-data#extraction

I don't recall what was the last one that was used for blocks. 

It would be good to add that information in https://github.com/PrismarineJS/minecraft-data/blob/master/doc/add-data-new-version.md once we get it

In the mean time, can you still open the PR ?

We will figure out how to add back the missing data there

**Comment by Moondarker** [comment]
Alright, I'll open a PR with changes for 1.12 for now and will add the remaining versions once we figure out everything else

**Comment by Moondarker** [comment]
TL;DR: If you extract data for >=1.13 it will look exactly like in https://github.com/PrismarineJS/minecraft-data/pull/384, as for versions <=1.12, I should do some consistency checks before commiting bare results of running burger-extractor

**Review by rom1504 — APPROVED** [review]
